subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/TestRunEnvironment.kt

```diff
@@ -96,14 +99,27 @@ fun provideEnvironment(
     return when (isFakeOrchestratorRun) {
         true -> TestRunEnvironment.OrchestratorFakeRunEnvironment
         false -> try {
+            val coordinates = ReportCoordinates(
+                planSlug = argumentsProvider.getMandatoryArgument("planSlug"),
+                jobSlug = argumentsProvider.getMandatoryArgument("jobSlug"),
+                runId = argumentsProvider.getMandatoryArgument("runId")
+            )
+            // todo move to instrumentation plugin
```

Comment:
local report will always be enabled?

Reply:
If you have url u can enable it. If you don't - can disable